### ISSUE TITLE
Amiga CD32 Games wont start anymore

### ISSUE THREAD

**Issue by Retro1968** [midwan/amiberry#678]
Raspberry Pi 4 with RetroPie 4.6.5 and amiberry 3.2 beta (2020-08-05).

My AmigaCD 32 games (.bin +.cue) wont start anymore.
Only the Kickrom 1.3 screen appears.

Cheers
Retro1968

**Comment by midwan** [comment]
This is "expected behavior" due to the changed cmd line parameter syntax.

There's an open issue with retropie to update their launch script, regarding this. One that is done, it should be just a matter of updating the retropie script and it will work again.

**Comment by Retro1968** [comment]
And what exactly is to be done now?

**Comment by Retro1968** [comment]
Bevorstehende Änderungen am Verlauf
Der Übersetzungsverlauf steht bald nur noch zur Verfügung, wenn du angemeldet bist. Er wird dann zentral über Meine Aktivitäten verwaltet. Der bisherige Verlauf wird durch dieses Upgrade gelöscht. Speichere also Übersetzungen, auf die du später noch zugreifen möchtest.
OK
843/5000
I used my backup with RetroPie 4.6.2 again, because EVERYTHING is going well.

A Linux update recently caused the screen resolutions to appear to have been redistributed. Before, e.g. 52-11 = 1280x1024, after that it was 1920x1080. The PSX emulator also stuttered afterwards.

RetroPie is now introducing an "improvement" that I clearly see as a disadvantage because the start scripts have apparently been changed.

Some may certainly see an advantage in this and claim that developments are necessary in order to achieve improvements.

Of course, none of this is your fault, Midwan. You are doing a fantastic job and you offer great support.

I have made up my mind to no longer perform Linux and RetroPie updates, but only to update the desired emulators.

Cheers
Retro1968
